### ISSUE TITLE
Parse error when upgrading from v3.6.5 to v3.6.6

### ISSUE THREAD

**Issue by jeremyf** [seattlerb/ruby_parser#183]
I'm running brakeman against an existing project. For the following line [from an ERB template](https://github.com/ndlib/sipity/blob/5fd618993f811960287f010dcc392fd182a80dcd/app/views/sipity/controllers/account_profiles/edit.html.erb#L1)

```erb
<%= content_tag :h1, t("sipity/decorators/account.call_to_action"), class: 'centered with-bottom-padding' %>
```

I am getting the following error in v3.6.6 of ruby_parser (brakeman v3.0.2). 

```console
./sipity/app/views/sipity/controllers/account_profiles/edit.html.erb:1 :: parse error on value ":" (tCOLON) | 
Could not parse ./sipity/app/views/sipity/controllers/account_profiles/edit.html.erb
```

This showed up after updating from v3.6.5 of ruby_parser.

The following files are in error:

| Error | Location |
|-----------------------|-----------------------|
| ./sipity/app/models/sipity/models/account_placeholder.rb:14 :: parse error on value ":" (tCOLON)            | Could not parse ./sipity/app/models/sipity/models/account_placeholder.rb             |
| ./sipity/app/models/sipity/models/collaborator.rb:17 :: parse error on value ":" (tCOLON)                   | Could not parse ./sipity/app/models/sipity/models/collaborator.rb                    |
| ./sipity/app/models/sipity/models/doi_creation_request.rb:7 :: parse error on value ":" (tCOLON)            | Could not parse ./sipity/app/models/sipity/models/doi_creation_request.rb            |
| ./sipity/app/models/sipity/models/work.rb:11 :: parse error on value ":" (tCOLON)                           | Could not parse ./sipity/app/models/sipity/models/work.rb                            |
| ./sipity/app/views/sipity/controllers/account_profiles/edit.html.erb:1 :: parse error on value ":" (tCOLON) | Could not parse ./sipity/app/views/sipity/controllers/account_profiles/edit.html.erb |

**Comment by jeremyf** [comment]
The travis build environment https://travis-ci.org/ndlib/sipity/jobs/58432807

**Comment by jeremyf** [comment]
Fixed via @b025dc85c7ee838906fb44536b0da2d2d9bf1ef9
